test/models/discount.rb

```diff
@@ -0,0 +1,3 @@
+class Discount < ActiveRecord::Base
+  belongs_to :discountable, polymorphic: true
```

Comment:
Thank you for adding a test and updating the test fixtures! This makes it so much easier to review the changes.